BIPs/BIP-162/arbitrumTreasury.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1674765144498,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.11.1",
+    "createdFromSafeAddress": "0xaF23DC5983230E9eEAf93280e312e57539D098D0",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xb029cae52cf9d55447a8f7d9abe8550e224652e1362d5c89fd6997ea9dd7fafb"
+  },
+  "transactions": [
+    {
+      "to": "0x64541216bAFFFEec8ea535BB71Fbc927831d0595",
```

Comment:
The DAO Treasury on Arbitrum at arb1:0xaF23DC5983230E9eEAf93280e312e57539D098D0 will initiate a transfer of “B-33WETH-33WBTC-33USDC” (contract 0x64541216bAFFFEec8ea535BB71Fbc927831d0595) in the amount of 532147981134526165239 to the LM Multisig at arb1:0xc38c5f97B34E175FFd35407fc91a937300E33860. From there, BPT will be withdrawn and funds bridged back to Ethereum. All funds will be sent to the Karpatkey safe 0x0EFcCBb9E2C09Ea29551879bd9Da32362b32fc89